app/services/vacancy_filters.rb

```diff
@@ -20,7 +20,7 @@ def initialize(args)
   def to_hash
     {
       location: location,
-      radius: "#{radius}km",
```

Comment:
Also, `miles` is the unit used in the front end. Hmm.

Reply:
This is because I used kilometers in the [initial geocoder work](https://github.com/dxw/teacher-vacancy-service/commit/a09a2fbc33614132135608495e28b8f00b6648a8).

However I can't figure out if `to_hash` was used anywhere (before being plucked into the AuditSearchEvent job), which might be why we never realised.

Related commit: https://github.com/dxw/teacher-vacancy-service/commit/1d96e1c59dbaad8f43379f876c19eb556e5dbf80